crates/adapters/src/adhoc/table.rs

```diff
@@ -80,6 +80,10 @@ pub struct AdHocTable {
     input_handle: Option<Box<dyn DeCollectionHandle>>,
     name: SqlIdentifier,
     materialized: bool,
+    /// True if the table is indexed.
```

Comment:
at least technically indexes can only apply to views in our SQL. maybe this needs a little explanation